despatch_advice_import_ubl/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Despatch Advice Import Ubl
+==========================
+
+Import Despatch Advice files
```

Comment:
In fact there 2 workflow with despatch:

1. yours: your receive file about goods that'll receive soon
2. other flow: you define a delivery order for your customer (picking out is created at sale confirmation) and your warehouse site/manager send you a file to indicate which lots are really sent, then have to match with an existing picking

Probably interesting to define here that you only implements the first one. Thanks

Reply:
I have to implement the last in v12, for information

Reply:
Also a despatch, couldn't be linked to an order (sale or purchase) even if UBL require it.
In that case, we only deal with picking, for example between warehouses

Reply:
Please @lmignon @lmarion-source are you OK with the different possible use cases ? Sure I don't want you implement them here. Just I would know if I'm not in error functionally speaking. Thanks